### PR TITLE
[chore] enable ARM64 linux package tests

### PR DESCRIPTION
**Pull request by mowies**
This PR adds linux package tests for arm64 .deb and .rpm packages using QEMU

### CODE REVIEW

File: internal/testbed/linux-services/package-tests.sh [hunk 1]
```diff
@@ -28,6 +29,13 @@ if [[ ! -f "$PKG_PATH" ]]; then
     exit 1
 fi
 
+# translate distro x86 arch to normal name
```
Comment (pirgeo) [inline internal/testbed/linux-services/package-tests.sh:32]:
```suggestion
# translate distro x86 arch to podman arch name
```

Comment (mowies) [reply internal/testbed/linux-services/package-tests.sh:32]: but it is the generally used name i believe 😅

File: .github/workflows/snapshot.yaml [hunk 2]
```diff
@@ -56,10 +55,14 @@ jobs:
...
         os: ["deb", "rpm"]
-        arch: ["x86_64"] # "arm64"] enable ARM when github hosted ARM runners are available
+        arch: ["x86_64", "arm64"]
     steps:
       - name: Checkout
         uses: actions/checkout@v4
+      - uses: docker/setup-qemu-action@v3
+        with:
+          platforms: arm64
+      - uses: docker/setup-buildx-action@v3
```
[the remark this answers is not in the document]
Comment (mowies) [reply .github/workflows/snapshot.yaml:65]: no, this should work fully transparent. Also, i specifically only setup qemu for `arm64` in line 64 here.

File: internal/testbed/linux-services/package-tests.sh (hunk 1 above)
Comment (pirgeo) [reply internal/testbed/linux-services/package-tests.sh:32]: Oh, okay, I guess I am not spending enough time with folks who are discussing arch enough, I guess I am just not familiar with it 😅

Comment (mowies) [reply internal/testbed/linux-services/package-tests.sh:32]: at least goreleaser uses `amd64` as default and we override it 😄

File: .github/workflows/snapshot.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (pirgeo) [reply .github/workflows/snapshot.yaml:65]: Ah I guess I should look a bit more closely  🦇

**Pull request merged**